app/views/transactions/create.js.erb

```diff
@@ -1,7 +1,9 @@
-<% if @transaction.persisted? %>
-  $("<%=j render('transactions/transaction', transaction: @transaction) %>").hide().prependTo(".transactions").fadeIn(1000);
-  $("#new_transaction").replaceWith("<%=j render('transactions/form', transaction: Transaction.new(category_id: params[:transaction][:fixed_category_id]), fixed_category_id: params[:transaction][:fixed_category_id]) %>");
-  scrollTo($("#<%= dom_id(@transaction) %>"));
-<% else %>
-  $("#new_transaction").replaceWith("<%=j render('transactions/form', transaction: @transaction, fixed_category_id: params[:transaction][:fixed_category_id]) %>");
+<% if @transaction %>
```

Comment:
@Gataniel why do you need this? This view should be rendered without `@transaction` variable. If it's rendered some way then something goes wrong.